test/framework/src/main/java/org/elasticsearch/action/admin/cluster/reroute/ClusterRerouteUtils.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.action.admin.cluster.reroute;
+
+import org.elasticsearch.ElasticsearchException;
+import org.elasticsearch.action.support.ActionTestUtils;
+import org.elasticsearch.action.support.SubscribableListener;
+import org.elasticsearch.client.internal.ElasticsearchClient;
+import org.elasticsearch.cluster.routing.allocation.command.AllocationCommand;
+
+import static org.elasticsearch.test.ESTestCase.asInstanceOf;
+import static org.elasticsearch.test.ESTestCase.safeAwait;
+import static org.elasticsearch.test.ESTestCase.safeGet;
+import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked;
+
+public class ClusterRerouteUtils {
+    private ClusterRerouteUtils() {/* no instances */}
+
+    public static void reroute(ElasticsearchClient client, AllocationCommand... allocationCommands) {
```

Comment:
Similarly, "Runs an empty reroute (TransportClusterRerouteAction) command. Optionally can supply a list of {@link AllocationCommand}s"

Reply:
++ done in 5a826956204ec968de2bf7a8e95ae3e02e9b1d0e.